package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@adobe/aio-cli-plugin-api-mesh",
-	"version": "4.1.0",
+	"version": "4.1.0-beta.7",
```

Comment:
I figure you're just testing, but would recommend setting this to `4.1.1-beta` now that `4.1.0` is published.

Reply:
Makes sense, done